_posts/2023-11-22-privacy-preserving-scikit-learn.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "End-to-end privacy for scikit-learn model training and inference"
+date: November 22, 2023
+
+#### Post Category and Tags ####
+# Format in titlecase without dashes (Ex. "Open Source" instead of "open-source")
+categories:
+  - Updates
+tags:
+  - Open Source
+  - Machine Learning
+  - Privacy Preserving Machine Learning
+  - Fully Homomorphic Encryption
+  - Federated Learning
+  - Privacy
+
+#### Featured Image ####
+featured-image: privacy-preserving-ml.jpg
```

Comment:
not sure how it will look at the end, I was just able to see the .md
if you want some edits on the image, you'll tell us: in particular, we can add the scikit-learn logo inside the image, if you like it